integrated_channels/utils.py

```diff
@@ -221,3 +224,17 @@ def get_subjects_from_content_metadata(content_metadata_item):
             subjects.add(subject_name)
 
     return list(subjects)
+
+
+def generate_formatted_log(message, channel_name=None, enterprise_customer_identifier=None, is_error=False):
+    """
+    Formats and logs a message for the integrated channels.
+    """
+    if enterprise_customer_identifier:
+        enterprise_customer_identifier = '[ENT CUSTOMER: ' + enterprise_customer_identifier + '] '
+
+    if channel_name:
+        channel_name = '[' + channel_name + '] '
+
+    log_message = '[Integrated Channel] ' + enterprise_customer_identifier + channel_name + message
+    LOGGER.error(log_message) if is_error else LOGGER.info(log_message)  # pylint: disable=expression-not-assigned
```

Comment:
I think this is a great improvement indeed! have some other thoughts for later logging work but this should get us a great set of logs for now!